README.md

```diff
@@ -14,7 +14,7 @@ A multi platform watch companion app for Pebble/RebbleOS devices
     GITHUB_TOKEN=<GENERATED TOKEN>
     ```
 
-4. Install flutter on your machine. To make builds reproducible, we use exact flutter version in pubspec.yml. Thus we recommend you use [FVM](https://fvm.app/docs/getting_started/installation) to install flutter. After you install FVM, just run `fvm install` command in the
+4. Install flutter on your machine. To make builds reproducible, we use exact flutter version in pubspec.yml. Thus we recommend you use [FVM](https://fvm.app/documentation/getting-started/installation) to install flutter. After you install FVM, just run `fvm install` command in the
  project folder and you will automatically get the required flutter version. 
 5. [Setup flutter in the IDE of your choice](https://flutter.dev/docs/get-started/editor). Be sure to also configure it with [FVM Flutter path](https://fvm.app/docs/getting_started/configuration#ide).
```

Comment:
`https://fvm.app/docs/getting_started/configuration#ide` also doesn't work anymore but I'm not super clear on what the new page it should point to. I'm happy to also update it if someone can provide a new link 👍